frontend/src/app/home/home.page.html

```diff
@@ -7,18 +7,15 @@
           d="M390.307,55.034V69.156H376.025C376.025,62.566,383.007,55.034,390.307,55.034Zm0,30.965V71.877H376.025C376.025,78.467,383.007,86,390.307,86Zm2.751,0V71.877H407.34C407.34,78.467,400.357,86,393.058,86Z"
           transform="translate(-375.5, -55)"
         ></path>
-
         <path
           class="logo-square"
           d="M393.062,55.031h14.282V69.125H393.062V55.031Z"
           transform="translate(-375.5, -55)"
         ></path>
       </svg>
-
       <h1>Welcome to Consileon</h1>
       <h3>We make the difference</h3>
     </div>
-
     <div class="button-container">
       <button class="fancy-button" (click)="startApplication()">
         Apply now
```

Comment:
I don't see any reason why you have modified this file... (Also, I believe having some blank lines between the items makes it better to read.)

Reply:
Am not sure why is this file modified. I had checked this file for reference. will revert changes.